app/client/src/features/events/Questions/QuestionActions/QuestionActions.tsx

```diff
@@ -1,42 +1,57 @@
+import React from 'react';
 import { CardActions, CardActionsProps } from '@mui/material';
+import makeStyles from '@mui/styles/makeStyles';
 import { graphql, useFragment } from 'react-relay';
 
 import type { QuestionActionsFragment$key } from '@local/__generated__/QuestionActionsFragment.graphql';
 import { Like } from './Like';
 import { Quote } from './Quote';
 import { QueueButton } from './QueueButton';
+import { DeleteButton } from './DeleteButton';
+
+const useStyles = makeStyles(() => ({
+    actions: {
+        display: 'flex',
+        justifyContent: 'space-between',
+    },
```

Comment:
Were they not formatted correctly before?  I ask cause I remember adding styles specifically to make sure they were spaced appropriately, but maybe that was removed in a previous PR.

Reply:
It was only formatted correctly for like and quote, but when I added Delete it was overlapping so I added that to fix it.

Reply:
I'll take a closer look

Reply:
Resolving since we established that the component needs a little bit of cleanup and is out of scope for this PR.